radio.liq

```diff
@@ -114,11 +115,13 @@ def log_metadata_async(m) =
 end
 
 # create underwriting queue - if API is disabled, just leave it empty
-underwriting = if playlist_api_url != "" then
-    eat_blank(audio_to_stereo(request.dynamic.list(id="underwriting", retry_delay=900., underwriting_request)))
-else
-    empty(id="underwriting")
-end
+#underwriting = if playlist_api_url != "" then
+#    eat_blank(audio_to_stereo(request.dynamic.list(id="underwriting", retry_delay=900., underwriting_request)))
+#else
+#    empty(id="underwriting")
+#end
+
+underwriting = empty(id="underwriting")
```

Comment:
I think this actually needs to be a `request.queue` in order to push things to it via the Liquidsoap control port.

Reply:
yeah. I think you're right. Interestingly, we can tweak prefetch for queues; I didn't realise that (though it's not relevant here)